demos/bartlett1932/experiment.py

```diff
@@ -96,13 +96,12 @@ class Bot(BotBase):
     def participate(self):
         """Finish reading and send text"""
         try:
-            time.sleep(5)
-            self.driver.execute_script("create_agent_failsafe")
             logger.info("Entering participate method")
+            self.driver.execute_script("create_agent")
```

Comment:
I added that as a last ditch attempt to get it working before running out of time. The rationale was that the << loading >> text only goes away when that script is run, so for some reason it was not being run before we tried to get the textarea contents.